src/pages/Bid/BidDeadlineComponent.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+export default function BidDeadlineComponent() {
+  const [buttonDate, setButtonDate] = useState('');
+  //const [currentIdx, setCurrentIdx] = useState('')
+
+  const handleButtonForDeadline = event => {
+    const { name } = event.target;
+    switch (name) {
+      case '1':
+        setButtonDate(name);
+        break;
+      case '3':
+        setButtonDate(name);
+        break;
+      case '7':
+        setButtonDate(name);
+        break;
+      case '30':
+        setButtonDate(name);
+        break;
+      case '60':
+        setButtonDate(name);
+        break;
+      default:
+        break;
+    }
+  };
```

Comment:
- Switch 문은 불필요해 보입니다! 경우에 따라 다른 로직이 발생하는 것이 아니라, setButtonDate(name) 으로 동일한 로직이 실행되기 때문입니다.
```suggestion
  const handleButtonForDeadline = event => {
    const { name } = event.target;
    setButtonDate(name);
  };
```

Reply:
앗??? 정말 그렇네요 ?! 바로 수정했습니다